spec/lib/normalizr_spec.rb

```diff
@@ -119,5 +119,24 @@
         end
       end
     end
+    context 'normalizers are an array' do
+      let(:normalizers) { [[:strip, { truncate: { length: 8, omission: '...' }}, proc { |v| v.first(6) }]] }
```

Comment:
Hey @santi698, why you wrapped normalizers into another array? You have an array in an array here.